### PR TITLE
Home Page Launch

### PR DESCRIPTION
**Pull request by csell5**
Main theme was to convert from our event to a proper landing page. Also fixed a few minor items along the way.

* Spacing / Layout on Session Details screen
* Correlation ID now passed along with graph calls
* Home Page
** new sections include hero, stats, newsletter, events, cta, testimonials, 
** newsletter section is wired to AC
* Colors
* renamed ics download to match api download
*  added prodid to ics
* added sticky headers to main board.
* redid link color in footers
* fixed sizing on cards to take up max grid container.
* session details buttons align center on smaller screens
* bumped packages

### CODE REVIEW

File: src/components/home/Events.svelte [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                      more creative meaningful way, enabling our inner
+                      practitioner. Of course we can to do it anytime, anywhere,
+                      and any way we see fit.`,
+      url: '/events/thatOnline',
+      logoUrl: '/images/thatOnline.svg',
+    },
+    {
+      title: 'Day of the Dome',
+      date: 'October 17th, 2020',
+      description: `Cupim ham pastrami, chislic doner buffalo pig. Tri-tip pork chop andouille ham, corned beef filet mignon tail landjaeger sirloin chuck pork loin hamburger picanha. Meatball hamburger turkey frankfurter, porchetta chislic ham hock strip steak swine picanha alcatra tenderloin. Tongue flank fatback, leberkas cow jerky beef ribs. Leberkas filet mignon sausage meatball pork chop.`,
```
Comment (brettski) [inline src/components/home/Events.svelte:23]: I assume this is intentional and will be updated when the event is flushed out.

Comment (csell5) [reply src/components/home/Events.svelte:23]: yea. although I'm going to comment it out for now as to not pre announce something we haven't finalized.

Comment (csell5) [reply src/components/home/Events.svelte:23]: actually that whole array should call into the api and get the events.

File: src/components/home/Hero.svelte [hunk 2]
```diff
@@ -1,110 +1,222 @@
...
+
+              <p
+                class="text-base text-gray-500 sm:mt-5 sm:text-lg sm:max-w-xl
+                sm:mx-auto md:mt-5 md:text-xl lg:mx-0"
+              >
+                This is where THAT comes in. It’s our mission to connect you and
+                give you the safe place to have a conversation each and every
+                day. Let’s say you want to have a code review or a design
+                review, great. Maybe you want to share your experience with the
+                latest Javascript framework or seek some geek parenting advice.
```
Comment (brettski) [inline src/components/home/Hero.svelte:211]: I am pretty sure Javascript is **JavaScript**

**Review by brettski — DISMISSED** [review]

File: src/components/home/Hero.svelte (hunk 2 above)
Comment (brettski) [reply src/components/home/Hero.svelte:211]: @csell5 ☝️ 👆

Comment (csell5) [reply src/components/home/Hero.svelte:211]: LOL

**Comment by brettski** [comment]
By the way, this new home page is HOT 🔥

**Pull request merged**